### ISSUE TITLE
docker hub moqui/moquidemo image should not include elasticsearch

### ISSUE THREAD

**Issue by eigood** [moqui/moqui-framework#569]
Instead, it should contain opensearch.

**Comment by jonesde** [comment]
Agreed. The original build included OpenSearch, but the support for OpenSearch in the Dockerfile and related scripts was not complete yet and I decided not to do it in the middle of a release. Going forward it needs to be OpenSearch because ElasticSearch open source is end of life, but that's the main reason. Functionally in Moqui they are still the same (ie no OpenSearch features used that are not also supported in ElasticSearch). Overall it's a big flipping mess and it will unroll slowly, eventually maybe even to the point of renaming ElasticFacade to SearchFacade or something.

When you filed this issue, what was your goal or what did you want to see happen?

**Comment by eigood** [comment]
Just wanting to see the project stay open-source related. See my other
reply, which I haven't written yet
